test/remote/gateways/remote_d_local_test.rb

```diff
@@ -194,15 +194,9 @@ def test_successful_void
   end
 
   def test_failed_void
-    auth = @gateway.authorize(@amount, @credit_card, @options)
-    assert_success auth
-
-    assert capture = @gateway.capture(@amount, auth.authorization, @options)
-    assert_success capture
-
-    response = @gateway.void(auth.authorization)
```

Comment:
This failure case is not longer true. In this updated flow, DLocal will allow a void of the authorization that has already been captured. They are aware of this issue, but I decided it was low risk.